app/src/main/java/com/kickstarter/viewmodels/SearchViewModel.java

```diff
@@ -24,93 +24,131 @@
 import rx.subjects.BehaviorSubject;
 import rx.subjects.PublishSubject;
 
-public final class SearchViewModel extends ActivityViewModel<SearchActivity> implements SearchViewModelInputs, SearchViewModelOutputs {
-  // INPUTS
-  private final PublishSubject<String> search = PublishSubject.create();
-  @Override public void search(final @NonNull String s) {
-    search.onNext(s);
-  }
-  private final PublishSubject<Void> nextPage = PublishSubject.create();
-  public void nextPage() {
-    nextPage.onNext(null);
-  }
+public interface SearchViewModel {
 
-  // OUTPUTS
-  private final BehaviorSubject<List<Project>> popularProjects = BehaviorSubject.create();
-  @Override public Observable<List<Project>> popularProjects() {
-    return popularProjects;
-  }
-  private final BehaviorSubject<List<Project>> searchProjects = BehaviorSubject.create();
-  @Override public Observable<List<Project>> searchProjects() {
-    return searchProjects;
+  interface Inputs {
+    /**
```

Comment:
to keep these files shorter we like to do our block quote comment style as such:
```
/** Call when next page is invoked. */
```

you can refer to `ProfileViewModel` for the comments style, and this isn't merged in yet but here are some docs i've whipped up today: https://github.com/kickstarter/native-docs/pull/3